src/test/resources/org/prebid/server/it/openrtb2/krushmedia/test-logicad-bid-response.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "tid",
```

Comment:
it is a good practice to get this response from GO test version.

Reply:
Request is different, so response won't be same, but already updated request with data, which helps to check new headers